src/Functional.CQS.AOP.CommonTestInfrastructure/DummyObjects/DummyAsyncQueryReturnsReferenceTypeHandler.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System.Reflection;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Functional.CQS.AOP.CommonTestInfrastructure.DummyObjects
+{
+	// ReSharper disable once InconsistentNaming
```

Comment:
Out of curiosity, what was ReSharper complaining about here?

Reply:
Same question goes for `DummyAsyncQueryReturnsValueTypeHandler`.

Reply:
I probably had an underscore in the name at some point and then just neglected to delete the Resharper comment